client/sites/components/style.scss

```diff
@@ -301,10 +301,9 @@
 				max-height: calc(100vh - var(--masterbar-height));
 			}
 			.preview-pane__navigation {
-				box-shadow: 0 0 0 1px var(--color-border-subtle), 0 1px 2px var(--color-border-subtle);
-				// Fix the weird box-shadow on the top when the action buttons are wrapped into the newline;
+				box-shadow: none;
+				border-bottom: 1px solid var(--color-border-subtle);
```

Comment:
Is this of any issue to use `border` instead of `box-shadow`?
Use of box-shadow requires a higher `z-index` to avoid it falling behind other content, and might create other stacking problems. Hence I chose to switch to border instead.

Reply:
Can we check if this breaks the "fix" here https://github.com/Automattic/wp-calypso/pull/90622/files#r1598570785

Reply:
Verified the above issue. It is not observed in any of the viewports.